proposals/lip-0048.md

```diff
@@ -241,10 +252,11 @@ def verify(trs: Transaction) -> None:
     b = block including trs
```

Comment:
Comment about line 249: Perhaps add a sentence or two explaining this? In particular, that this module runs this step before *every* transaction submitted in the chain (perhaps link to LIP55 defining the steps..)

Reply:
Minor point: I meant to add it as text between subsection title and pseudocode. THis way you could put also a link to LIP 55).